docs/en/install-upgrade/installing-stack-demo-secure.asciidoc

```diff
@@ -0,0 +1,894 @@
+
+* <<install-stack-demo-secure-prereqs>>
+* <<install-stack-demo-secure-ca>>
+* <<install-stack-demo-secure-transport>>
+* <<install-stack-demo-secure-http>>
+* <<install-stack-demo-secure-second-node>>
+* <<install-stack-demo-secure-kib-es>>
+* <<install-stack-demo-secure-fleet>>
+* <<install-stack-demo-secure-agent>>
+* <<install-stack-demo-secure-view-data>>
+
+It should take between one and two hours to complete these steps.
+
+[discrete]
+[[install-stack-demo-secure-prereqs]]
+== Prerequisites and assumptions
+
+Before starting, you'll need to have set up an on-premises {es} cluster with {kib}, following the steps in <<installing-stack-demo-self>>.
+
+The examples in this guide use RPM packages to install the {stack} components on hosts running Red Hat Enterprise Linux 8. The steps for other install methods and operating systems are similar, and can be found in the documentation linked from each section.
+
+Special considerations such as firewalls and proxy servers are not covered here.
+
+[discrete]
+[[install-stack-demo-secure-ca]]
+== Step 1: Generate a new self-signed CA certificate
+
+In a production environment you would typically use the CA certificate from your own organization, along with the certificate files generated for the hosts where the {stack} is being installed. For demonstration purposes, we'll use the Elastic certificate utility to configure a self-signed CA certificate.
+
+. On the first node in your {es} cluster, stop the {es} service:
++
+["source","shell"]
+----
+sudo systemctl stop elasticsearch.service
+----
+
+. Generate a CA certificate using the provided certificate utility, `elasticsearch-certutil`. Note that the location of the utility depends on the installation method you used to install {es}. Refer to {ref}/certutil.html[elasticsearch-certutil] for the command details and to {ref}/update-node-certs-different.html[Update security certificates with a different CA] for details about the procedure as a whole.
++
+Run the following command. When prompted, specify a unique name for the output file, such as `elastic-stack-ca-new.zip`: 
++
+["source","shell"]
+----
+sudo /usr/share/elasticsearch/bin/elasticsearch-certutil ca -pem
+----
+
+. Move the output file to the `/etc/elasticsearch/certs` directory. This directory is created automatically when you install {es}.
++
+["source","shell"]
+----
+sudo mv /usr/share/elasticsearch/elastic-stack-ca-new.zip /etc/elasticsearch/certs/
+----
+
+. Change to the certificates directory and unzip the file:
++
+["source","shell"]
+----
+sudo cd /etc/elasticsearch/certs
```

Comment:
This won't work. The user would need to be root (sudo won't work for 'cd'-ing into a directory). I think we can remove this step and just use full paths for the following steps.